src/VisualStudio/Core/Def/Implementation/AnalyzerDependencyCheckingService.cs

```diff
@@ -46,7 +46,7 @@ internal sealed class AnalyzerDependencyCheckingService
             {
                 results = await GetConflictsAsync().ConfigureAwait(continueOnCapturedContext: true);
             }
-            catch (OperationCanceledException)
+            catch
```

Comment:
@tmeschter That would mean the following:

```csharp
catch (BadImageFormatException)
{
  // This can occur if someone passes a non-managed DLL to GetConflictsAsync.
  // While normally treated as a critical exception, we don't consider it critical
  // at this particular location.
  return;
}
catch (Exception ex)
{
  if (ErrorHandler.IsCritical(ex))
    throw;

  return;
}
```